pkg/controllers/default_resources.go

```diff
@@ -38,13 +38,13 @@ import (
 
 var resLog = ctrl.Log.WithName("resource-render")
 
-const ReleaseImage = "quay.io/openshift-release-dev/ocp-release:4.9.15-x86_64"
+const ReleaseImage = "quay.io/openshift-release-dev/ocp-release:4.10.6-x86_64"
 
 func getReleaseImagePullSpec() string {
 
 	defaultVersion, err := version.LookupDefaultOCPVersion()
 	if err != nil {
-		return "invalid-image.com"
+		return ReleaseImage
```

Comment:
Should we print a log here?

Reply:
the output is at

https://github.com/ianzhang366/hypershift-deployment-controller/blob/1ec35c5a30dff7ecfa5a624466bb14fe21ee9ecc/pkg/controllers/default_resources.go#L147

having a log is kind repetive